compiler/rustc_codegen_cranelift/src/vtable.rs

```diff
@@ -83,88 +83,15 @@ pub(crate) fn get_vtable<'tcx>(
 
 fn build_vtable<'tcx>(
```

Comment:
You can inline this function into `get_vtable` now that it is much smaller.

Reply:
Done.